bin/citgm-all.js

```diff
@@ -83,11 +88,13 @@ function runCitgm (mod, name, next) {
     process.removeListener('SIGINT', cleanup);
     process.removeListener('SIGHUP', cleanup);
     process.removeListener('SIGBREAK', cleanup);
+    process.setMaxListeners(process.getMaxListeners() - app.parallel || process.getMaxListeners());
```

Comment:
this could also cause weirdness with the number being lowered when one job finishes